### PR TITLE
Bael 2969

### PR DESCRIPTION
**Pull request by codehunter34**
BAEL-2969: Copying Sets in Java

### CODE REVIEW

**Comment by eugenp** [comment]
test again

File: core-java-modules/core-java-collections-set/pom.xml [hunk 1]
```diff
@@ -1,33 +1,46 @@
...
+	<name>core-java-collections-set</name>
+	<packaging>jar</packaging>
 
-    <parent>
-        <groupId>com.baeldung</groupId>
-        <artifactId>parent-java</artifactId>
-        <version>0.0.1-SNAPSHOT</version>
-        <relativePath>../../parent-java</relativePath>
-    </parent>
+	<parent>
```
Comment (rpvilao) [inline core-java-modules/core-java-collections-set/pom.xml]: wrong indentation, use 4 spaces

File: core-java-modules/core-java-collections-set/src/main/java/com/baeldung/set/CopySets.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            .collect(Collectors.toSet());
+
+        // Null check in case of expecting null values
+        Set<T> copy4 = original.stream()
+            .filter(Objects::nonNull)
+            .collect(Collectors.toSet());
+
+    }
+
+    public static <T> Set<T> copyByJava8(Set<T> original) {
```
Comment (rpvilao) [inline core-java-modules/core-java-collections-set/src/main/java/com/baeldung/set/CopySets.java]: as discussed with josh maybe we should rephrase it

**Comment by asif-anwar** [comment]
[retest this please]
Note: This comment is to trigger the Jenkins CI build

**Comment by rpvilao** [comment]
[retest this please]
Note: This comment is to trigger the Jenkins CI build

**Pull request merged**